static/js/hitchhikerconfigmanager.js

```diff
@@ -0,0 +1,55 @@
+import getHHConfigAttributes from './utils/getHHConfigAttributes';
+
+/**
+ * Responsible for managing the HitchhikerConfig object from the parent of an iframe
+ */
+export class HitchhikerConfigManager {
+  constructor () {
+    this._hitchhikerConfig = getHHConfigAttributes('#answers-container');
+  }
+
+  getObject () {
+    return { ...this._hitchhikerConfig };
+  }
+
+  set (key, value) {
+    this._validate(key, value);
+    this._hitchhikerConfig[key] = value;
+  }
+
+  sendConfigToIframe() {
```

Comment:
I think it might be a bit cleaner if we leave this logic to `iframe-common`. Without this method, we could just make this class `HitchhikerConfig.js`. It's purely a run-time data store. Consumers of the data store can add additional behaviors, such as sending the info to an iFrame via a message. In theory, someone could use this thing in the non-iFrame integration.

Reply:
sure, that sounds good